### PR TITLE
Karthik

### PR DESCRIPTION
**Pull request by tecnologist**
please find layout css added to attractions.html and cleared redudant code from attractions.css

### CODE REVIEW

File: css/attractions.css [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+
+/* Banner */
+
+#container{ 
+  padding: 0px;
+  position: relative;
+  text-align: center;
+  color: white;
+  font-family: "Times New Roman", Times, serif;
```
Comment (steve-human) [inline css/attractions.css:19]: I'm thinking we should have the font consistent throughout the site. Thoughts?

Comment (tecnologist) [reply css/attractions.css:19]: hi steve, I  changed font only for that text alone , as original font did not make the heading text appear distinct from its background banner image. rubric also states thoughtful and appropriate styling of text gives us more points, though I dont mind changing the font if you feel its best to keep it consistent. let me know

Comment (tecnologist) [reply css/attractions.css:19]: > I'm thinking we should have the font-family consistent throughout the site. The layout.css uses Roboto as the main font. Thoughts?


hi steve, I changed font only for that text alone , as original font did not make the heading text appear distinct from its background banner image. rubric also states thoughtful and appropriate styling of text gives us more points, though I dont mind changing the font if you feel its best to keep it consistent. let me know

Comment (steve-human) [reply css/attractions.css:19]: okay, let's keep it as is for now

**Pull request merged**